browser/scripts/ui-site.js

```diff
@@ -0,0 +1,581 @@
+
+	};
+
+	this.init = function() {
+		that.attach();
+
+		if (jQuery('body.bHome').length > 0) {
+			that.initHomepage();
+			signupCallback = function() {	// #slush
+				window.location.href = '/edit';
+			};
+		}
+
+	};
+
+	this.initHomepage = function($body) {
+		jQuery('a#homeSignin', $body).on('click', function(e){
+			e.preventDefault();
+			e.stopPropagation();
+			VizorUI.openLoginModal()
+			.then(function(){
+				document.location.href="/edit";
+			});
+			return false;
+		});
+
+		jQuery('a.readmore.mobileonly', $body)
+			.on('click', function(e){
+				e.preventDefault();
+				e.stopPropagation();
+				jQuery(this).addClass('used');
+				var anchor = '#'+this.href.split('#')[1];
+				jQuery(anchor)
+					.hide()
+					.removeClass('nomobile')
+					.css({margin: '0 0 10px 0'})
+					.slideDown('medium');
+				return false;
+			});
+
+		$('.team-member').click(function() {
+			window.open($(this).find('.profile-link').attr('href'));
+		})
+		$('.team-button').click(function(e) {
+			var teamY = $window.height();
+			$("html, body").animate({scrollTop: teamY}, 500);
+			e.preventDefault();
+			return false;
+		});
+
+		var $getStarted = jQuery('section#getstarted').first();
+		var onResize = VizorUI.makeVRCanvasResizeHandler(jQuery('#webgl-canvas'), $getStarted);
+		$(window).on('resize', onResize);
+		$(window).on('vizorLoaded', function() {
+
+			E2.app.isVRCameraActive = function(){return false};	// disable panning on homepage player, see #790
+			E2.app.calculateCanvasArea = function() {
+                return {
+                    width: $getStarted.innerWidth(),
+                    height: $getStarted.innerHeight()
+                }
+            }
+			onResize();
+		});
+
+		jQuery('button#mobileMenuOpenButton').on('mousedown touchdown', function(e){
+			e.preventDefault();
+			e.stopPropagation();
+			var $mobileMenu = jQuery(E2.views.partials.homeMobileMenu())
+				.appendTo('div#contentwrap');
+			var menuHeight = $mobileMenu.outerHeight();
+
+			var $body = jQuery('body');
+			$body.css({height:menuHeight, overflow:'hidden'});
+			$body.scrollTop(0);
+
+			var dismissMenu = function() {
+				$body.css({height:'', overflow:'initial'});
+				jQuery('#mobilemenu')
+					.fadeOut('fast', function(){
+						jQuery(this).remove();
+					});
+
+				return true;
+			}
+			jQuery('button#mobileMenuCloseButton', $mobileMenu).on('mousedown touchdown', function(e){
+				e.preventDefault();
+				e.stopPropagation();
+				dismissMenu();
+				return false;
+			});
+
+			$mobileMenu.fadeIn('fast');
+			jQuery('a', $mobileMenu).on('mousedown touchdown', dismissMenu)
+			VizorUI.enableScrollToLinks($mobileMenu);
+
+			return false;
+		});
+
+	}
+
+}
+
+jQuery('document').ready(siteUI.init);
+
+
+if (typeof VizorUI === 'undefined') var VizorUI = {};
+
+VizorUI.makeVRCanvasResizeHandler = function($playerCanvas, $containerRef) {
+	if (!($playerCanvas instanceof jQuery)) {
```

Comment:
usually past midnight, when one is bleary-eyed, i expect :)
also in all other times one accidentally passes just a documentElement (a `this` or `e.target` being the usual culprits)